app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -673,6 +673,73 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_withUser_forgot_inputAdminPinAndNullPin_errorIsDisplayed() {
```

Comment:
Also, add a configChange version on this test and separately test it.

Reply:
Done and updated the sc of passed tests.